src/map/atcommand.c

```diff
@@ -3501,7 +3501,7 @@ ACMD(spiritball)
 	if( sd->spiritball > 0 )
 		pc->delspiritball(sd, sd->spiritball, 1);
 	sd->spiritball = number;
-	clif->spiritball(&sd->bl, BALL_TYPE_SPIRIT, AREA);
+	clif->spiritball(&sd->bl, 1, AREA);
```

Comment:
I'm not really a fan of magic numbers, can you revert this to use an enum? It makes the code a lot easier to read

Reply:
this parameter set how many spirits should be spawned.
i don't think make any sense to use enum here.

Reply:
But then why does it say this

```
 * Ball Type:
 *      0 = None
 *      1 = Spirit
 *      2 = Soul
```

Reply:
ah here it not assigned directly to ``num`` then yes can be used enum